spec/opportunities/user_searches_for_opportunities_spec.rb

```diff
@@ -0,0 +1,49 @@
+feature "User searches for opportunities" do
+
+  let!(:user) { FactoryGirl.create(:user, volunteer: false) }
+  let!(:organization) { FactoryGirl.create(:organization) }
+  let!(:org_admin) { FactoryGirl.create(
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.